### PR TITLE
All_setup

### PR DESCRIPTION
**Pull request by Tooba-Amjad**
@VidhiBhatt01 
I simply copied the CSS code from the provided section and pasted it. Everything was set up as you requested, and I'm explaining this to you because you asked for some minor changes and requested that I create pull requests individually. However, it was all set up with just one click.

Screenshot (241)

Screenshot (242)

Screenshot (243)

### CODE REVIEW

**Comment by Tooba-Amjad** [comment]
@VidhiBhatt01 
This is how the complete Site Looks like now
Uploading WhatsApp Video 2023-10-30 at 6.59.50 PM.mp4…

**Comment by Reene-Simon** [comment]
@VidhiBhatt01 this has brought everything back to its original state. I'm currently working on some of the small details and will make a PR accordingly.

It was a change in the lower portion of the code with regards to the responsiveness which messed up with everything even in the upper part of the website.

File: style.css [hunk 1]
```diff
@@ -529,6 +529,11 @@ body {
   color: white; 
 }
 
+.social-icons i:hover {
```
Comment (VidhiBhatt01) [inline style.css:532]: this change should not be added in this commit.

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
Got it thanks for the effort girls. We are still missing some things.

1. Spacing between what do our students say.
2. Updating hover effect on LinkedIn icon.
3. Adding hover effect Rigorous Coding Practice, Regular Mock Interviews icons..


Also, how responsive is the site currently? Can you both make if fully responsive?

**Comment by VidhiBhatt01** [comment]
Okay, just saw the other PRs, ig we are sorted then. Fix LinkedIn hover effect and lmk about responsiveness

**Comment by Reene-Simon** [comment]
@VidhiBhatt01 the LinkedIn hover effect is being handled my Tooba and she has made a PR for the same. As far as responsiveness is concerned, we'll check it out and let uk.

**Comment by Reene-Simon** [comment]
@VidhiBhatt01 the responsiveness is mostly ok. Just want to check once this PR is merged as the layout is messed up as of now. After this PR is merged I'll give u an update about the same.

**Pull request closed without merging**